src/lib/Microsoft.Health.Events/EventCheckpointing/StorageCheckpointClient.cs

```diff
@@ -74,6 +75,8 @@ public async Task UpdateCheckpointAsync(Checkpoint checkpoint)
 
             try
             {
+                ct.ThrowIfCancellationRequested();
```

Comment:
If we keep passing the token into the below async methods do we need to explicitly call this?

Reply:
good point. I don't think so. removed it